### ISSUE TITLE
Membership rules, rights and obligations

### ISSUE THREAD

**Issue by taxmeifyoucan** [Bordel/bips#1]
Having a clear membership rules of hackerspace is key. Here are some basic rights and obligations to kick it off: 

1. Members are obligated to pay a monthly fee of 200k satoshi or equivalent in fiat shitcoin. (20€, 500czk)
2. Members must respect physical means of the space, other members and whole Paralelní Polis, be careful and do not misuse its property. Corporal punishment for thefts is opened for discussion, proposed by @JosefJ  
 _Don't hurt people and don't take their stuff._
3. Capacity of the hackerspace is limited to 20 members. 
4. Members of Bordel have access to the space and all of the tooling during opening hours of Paralelní Polis (weekdays 8am - 8pm) or when there are people with keys willing to open the space for them. Additionally, members will get access to selected events happening in Paralelni Polis. 
5. Members can propose an event they would like to officially organize in Bordel. Workshops, talks, discussions etc are welcomed. Sex orgies are opened for discussion. 
6. Each member can have an own box in hackerspace for storing his possessions. 
7. Unicorns are not prohibited but will be limited if space is overfilled with them. 

This list is opened for discussion and new rules that should have their own discussion can be added via its own issue.

**Comment by JosefJ** [comment]
Ok, this sounds legit. I'd add one more rule:

Since I have strong believe that it's the chaos that evolves from simple base order, I'd like to set a basis of maintaining a relative order in the space. Basically: 
"Leave the space in such (or better) condition you'd like to find in when you show up." 
This can obviously mean a lot of different things to different people, but I'm hopeful we all have some basic sense of maintaining elementary hygiene of our beloved Bordel. 

In other words: leave the tables and floors in a condition suitable for eating from them (even the floor, just don't actually do it).
